src/main/java/mServer/MServer.java

```diff
@@ -20,15 +20,20 @@
 package mServer;
 
 import java.io.File;
+
 import mServer.daten.MserverSearchTask;
 import mServer.search.MserverSearch;
-import mServer.tool.*;
+import mServer.tool.MserverDaten;
+import mServer.tool.MserverKonstanten;
+import mServer.tool.MserverLog;
+import mServer.tool.MserverTimer;
+import mServer.tool.MserverXmlLesen;
 import mServer.upload.MserverUpload;
 
 public class MServer {
 
     private MserverSearchTask aktSearchTask = null;
-    private boolean suchen = false;
+    public boolean suchen = false;
```

Comment:
Das ist dem anfänglichen Testen geschuldet und wurde meinerseits nur vergessen an der Stelle sauber in einer Getter zu packen. Werde ich bei Gelegenheit noch anpassen, darf aber auch gern jemand anders fix machen.